cdap-ui/app/cdap/components/Experiments/DetailedView/TopPanel/index.js

```diff
@@ -38,13 +38,13 @@ const Metadata = ({name, description, srcpath, total, outcome}) => {
       </div>
       <div>
         <strong>
-          <div> # Models </div>
+          <div> # of models </div>
           <h1>{total}</h1>
         </strong>
       </div>
       <div>
         <div>
-          <strong className="experiment-meta-label">Test Data</strong>
+          <strong className="experiment-meta-label">Data</strong>
```

Comment:
what does this fix?

Reply:
Nitin had a comment during one of my demos, saying that it should be 'Data' instead of 'Test Data'